learn/how-tos/azure-repos-vcs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Configure Azure Repos In CodeRepository"
+id: "azure-repos-vcs"
+---
+
+WaveMaker Team offering lets Team admins to configure the version control system where project source code is stored. From WaveMaker 11 GA version Team admins can configure Azure Repos under the Code Repository page. Although Azure contains Git and Tfvc repositories, WaveMaker supports Git Repositories. All the WaveMaker projects created in Azure Repos will be Git repositories.
```

Comment:
Do you mean Azure supports only Git repos, and Tfvc is not supported?

Reply:
Azure supports Both TFVC and Git But Wave Maker supports only Git

Reply:
> Do you mean Azure supports only Git repos, and Tfvc is not supported?

Azure supports both Tfvc and Git but Wave Maker supports only Git